src/services/__mocks__/productService.ts

```diff
@@ -213,19 +213,48 @@ export const mockedProductRoles: Array<ProductRole> = [
   },
 ];
 
-export const mockedCategories = {
-  product: {
-    'prod-pn': {
-      ipa: {
-        PA: 'L6,L4,L45,L35,L5,L17,L15,C14',
-      },
+export const mockedCategories: ProductInstitutionMap = {
+  'prod-interop': {
+    PA: {},
+    GSP: {},
+    SA: {},
+    SCP: {},
+    AS: {},
+    PRV: {},
+  },
+  'prod-pn': {
+    PA: {
+      categories: 'L6,L4,L45,L35,L5,L17,L15,L7,L22',

```

Comment:
categories handled as comma-separated strings can be error-prone, especially when checking for the existence of a category using includes....you could Convert the comma-separated category strings into arrays for accurate and efficient lookups, for ex: categories: ['L6', 'L4', 'L45', 'L35', 'L5', 'L17', 'L15', 'L7', 'L22']

Reply:
This is only mock data to simulate the API response. The onboarding FE needs to send a coma separeted string when filtering instituions. In the productFilters.ts file, I transform this string into an array before verifying if categoryCode in included in it.